.github/workflows/on-pull-request.yml

```diff
@@ -11,11 +11,20 @@ env:
 
 jobs:
   rustfmt:
-    name: Style & Lint
+    name: main - Style & Lint
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
 
+      - name: Verify README generation
+        uses: momentohq/standards-and-practices/github-actions/oss-readme-template@gh-actions-v2
+        with:
+          project_status: official
+          project_stability: alpha
```

Comment:
This is beta in the generate readme section.

Reply:
oops. will change it to alpha.